src/components/ReportDetailed/index.js

```diff
@@ -19,13 +21,27 @@ type ReportDetailedType = {
 
 type ReportDetailedState = {
   opened: boolean,
+  printable: any, 
```

Comment:
This actually has a shape:

```
printable: {
  date: boolean;
  start: boolean;
}
```

And the rest of the fields.

It's better to be as detailed as possible about the shape of objects in the store